db/seeds.rb

```diff
@@ -1,153 +1,153 @@
 Admin.create([{ first_name: "Admin_first",
-								last_name: "Admin_last",
-								email: "admin@email.com",
-								password: "password"
-								}])
+                last_name: "Admin_last",
+                email: "admin@email.com",
+                password: "password"
+                }])
 
 User.create([{ first_name: "Alice",
-							 last_name: "alast_name",
-						   email: "alice@email.com",
-						   display_name: "user_1",
-						   password: "password"},
-						  {first_name: "Bob",
-						   last_name: "blast_name",
-						   email: "bob@email.com",
-						   display_name: "user_2",
-						   password: "password"},
-						  {first_name: "Carol",
-						   last_name: "clast_name",
-						   email: "carlo@email.com",
-						   display_name: "user_3",
-						   password: "password"},
-						  {first_name: "Dave",
-						   last_name: "dlast_name",
-						   email: "dave@email.com",
-						   display_name: "user_4",
-						   password: "password"},
-						  {first_name: "eve",
-						   last_name: "elast_name",
-						   email: "eve@email.com",
-						   display_name: "user_5",
-						   password: "password"
-						   }])
+               last_name: "alast_name",
+               email: "alice@email.com",
+               display_name: "user_1",
+               password: "password"},
+              {first_name: "Bob",
+               last_name: "blast_name",
+               email: "bob@email.com",
+               display_name: "user_2",
+               password: "password"},
+              {first_name: "Carol",
+               last_name: "clast_name",
+               email: "carlo@email.com",
+               display_name: "user_3",
+               password: "password"},
+              {first_name: "Dave",
+               last_name: "dlast_name",
+               email: "dave@email.com",
+               display_name: "user_4",
+               password: "password"},
+              {first_name: "eve",
```

Comment:
Align the elements of an array literal if they span more than one line.<br>Space inside { missing.